skema/text_reading/scala/src/main/scala/org/ml4ai/skema/text_reading/grounding/scala_grounders/Adapter.scala

```diff
@@ -0,0 +1,82 @@
+package org.ml4ai.skema.grounding.scala_grounders
+
+import org.clulab.scala_grounders.grounding.GroundingConfig
+import org.ml4ai.skema.text_reading.grounding.Grounder
+import org.ml4ai.skema.text_reading.grounding.GroundingCandidate
+import org.ml4ai.skema.text_reading.grounding.GroundingConcept
+import com.typesafe.config.Config
+import org.clulab.scala_grounders.grounding.SequentialGrounder
+import org.clulab.scala_grounders.model.DKG
+import org.clulab.scala_grounders.model.DKGSynonym
+import com.typesafe.config.ConfigFactory
+import org.clulab.scala_grounders.using
```

Comment:
Recently we've been using the the `using` from
```
"org.scala-lang.modules" %% "scala-collection-compat" % version
```
often looking like `Using.resource() { resource => `.  It's in the standard library for newer Scala versions.